src/metabase/lib/drill_thru/quick_filter.cljc

```diff
@@ -65,8 +65,11 @@
       []
 
       (= value :null)
-      [{:name "=" :filter (operator :is-null  field-ref)}
-       {:name "≠" :filter (operator :not-null field-ref)}]
+      (for [[op label] (if (or (lib.types.isa/string? column) (lib.types.isa/string-like? column))
```

Comment:
The FE checks for both `string` and `string-like` types to render the UI, so we should match this behavior here